### DISCUSSION TITLE
在 "@antv/g2": "~4.2.10" 版本使用 Chart 时，请问如何取消 legend 的默认点击行为，即不想点击图例后，对应的折线消失，图例只用作显示

### DISCUSSION THREAD

**Discussion by SnowBeforeSpring** [antvis/G2#6281] [Q&A]
### 问题描述

import { Chart } from '@antv/g2'

  const res = new Chart({
    container: antVLineContainer.current as HTMLElement,
    autoFit: true,
  })

 chart.legend({
    position: 'top',
    marker: { symbol: 'hyphen' },
    // 这里看类型定义里面没有禁止点击事件的方法 or 属性
  })

### 平台

- 操作系统: [macOS, Windows, Linux, React Native ...]
- 网页浏览器: [Google Chrome, Safari, Firefox]

**Comment by pearmini** [comment]
可以看看文档，关闭对应的 interaction。
